clustered/client/src/main/java/org/ehcache/clustered/client/internal/store/operations/ChainResolver.java

```diff
@@ -56,68 +57,70 @@ public ChainResolver(final OperationsCodec<K, V> codec) {
    *
    * @param chain a heterogeneous {@code Chain}
    * @param key a key
-   * @param now time when the chain is being resolved
    * @return a resolved chain, result of resolution of chain provided
    */
-  public ResolvedChain<K, V> resolve(Chain chain, K key, long now) {
+  public abstract ValueHolder<V> resolve(ServerStoreProxy.Entry entry, K key, long now, int threshold);
```

Comment:
The javadoc is messed up, you at least need to change the `param` and `return` tags.

Reply:
Fixed.